mzLib/FlashLFQ/IsoTracker/PeakRegion.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+namespace FlashLFQ.IsoTracker
+{
+    /// <summary>
+    /// An PeakRegion should contain Apex Retention Time, Start and End Retention Time in the region.
+    /// </summary>
+    public class PeakRegion
+    {
+        public double ApexRT { get; private set; }
+        public double StartRT { get; private set; }
```

Comment:
You're using PeakRegion as a key for a dictionary. So you need to make sure that this class implements Equals and GetHashcode, and that the results from both methods agree. i.e.,  that Equal object have the same Hashcode and vice versa